README.md

```diff
@@ -1 +1,153 @@
+- **Meals info:** The API provides endpoints that retrieve details about meals, including image, category, ingredients, and instructions.
+- **API's USED**
+  - **[Meals API](https://www.themealdb.com/api/json/v1/1/filter.php?c=Chicken)** - for retrieving and displaying meals on the UI
+  - **[Interactive API](https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/9vUKLfgfPbeVlsgu5dzp)** - for user interactions such as likes and comments.
+
+<!-- GETTING STARTED -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming Soon
+
+## 🚀 Zoom Video (explanation) <a name="zoom"></a>
+
+[Author's Explanation](https://drive.google.com/file/d/1rSGYnAz1xglacDXcPURwLSgECzQ9TU28/view?usp=drive_link)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Web browser.
+- Code editor.
+- Git scm.
+
+Run the following commands, 
+in order to run this project: - `npm install`
+
+For tests `npm i jest-environment-jsdom --save-dev ` and ` npm run test`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/kessie2862/JavascriptCapstone.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd JavaScriptCapstone
+  npm install
+```
+
+### Usage
+
+To use this project - Run `npm run start` or `npm run build` or 
+Open `index.html` from `./dist` with your prefered browser.
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Prosper Kessie**
+
+- GitHub: [kessie2862](https://github.com/kessie2862)
+- Twitter: [Peekay_Codes](https://twitter.com/Peekay_Codes)
+- LinkedIn: [Prosper Kessie](https://www.linkedin.com/in/prosper-kessie-363968171/)
+
+👤 **Evans Karanja**
+
+- GitHub: [prg-04](https://github.com/prg-04)
+- Twitter: [Evans1425Sean](https://twitter.com/Evans1425Sean)
+- LinkedIn: [Evans Karanja](https://www.linkedin.com/in/evanson-karanja-3549841b8/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Implement recipe and contact pages**
+- [ ] **Add media query to make the page responsive**
+- [ ] **Enable sorting and filtering**
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/yordinia/JavaScript-Capstone/issues).
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give a star ⭐️ to this project. You can follow to see future projects.
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I'd like to thank Microverse and coding partner for his awesome contribution.
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/kessie2862/portfolio/blob/mobile/LICENSE.md) licensed.
```

Comment:
- [ ] 🚧 You did a great job documenting your project! 💯 However please make sure that you have a LICENSE and that its respective link in the README is working. Currently, this link leads to a different repo. You can follow these steps from [GitHub's official docs](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to set up a LICENSE. Microverse suggests using MIT! 🤓 🔧